pact/verifier.py

```diff
@@ -28,68 +28,160 @@ def __str__(self):
         return 'Verifier for {} with url {}'.format(self.provider, self.provider_base_url)
 
     def validate_publish(self, **kwargs):
-        """Validate publish has a version."""
-        if (kwargs.get('publish') is not None) and (kwargs.get('publish_version') is None):
-            # do something
-            raise Exception()
-
-    def verify_pacts(self, *pacts, enable_pending=False, include_wip_pacts_since=None, **kwargs):
-        """Verify our pacts from the provider.
-
-        Returns:
-          success: True if no failures
-          logs: some tbd output of logs
+        """Ensure the participant version is provided.
 
+        :raises Exception: When `publish_verification_results` is set,
+            but `publish_version` is not provided.
+        """
+        if not kwargs.get('publish_verification_results'):
+            return
+
+        if not kwargs.get('publish_version'):
+            raise Exception('To publish the results of verification to the '
+                            'broker version of the participant is mandatory')
```

Comment:
What about something like
```suggestion
    raise Exception('To publish the results of verification to the broker, '
                    'attribute publish_version (containing the provider '
                    'version) must be set')
```
I find it helpful if the error tells me what I need to set, so at least if I'm not sure where/how I have something to search for 🙂